Sources/Widgets/Representations/ArrowHandleRepresentation/index.js

```diff
@@ -168,6 +250,40 @@ function vtkArrowHandleRepresentation(publicAPI, model) {
     model.internalPolyData.modified();
     outData[0] = model.internalPolyData;
   };
+
+  publicAPI.requestData = (inData, outData) => {
+    const currShape = publicAPI
+      .getRepresentationStates(inData[0])[0]
+      .getShape();
+    if (shapeName !== currShape) {
+      if (firstShape === false) {
+        firstShape = true;
+      } else {
+        model.orientation = [1, 0, 0];
+        model.viewMatrix = mat4.create();
+      }
+      shapeName = currShape;
+      model.pipelines.arrow.glyph = detectArrowShape(shapeName);
+      model.glyph = model.pipelines.arrow.glyph;
```

Comment:
`model.glyph` was used to be a shortcut to `model.pipelines.arrow.glyph`.
Let's get rid of it because it does not help readability

Reply:
after some tests it seems that model.glyph is used to link glyph to the setGet glyph below, which i'm using to orient the cone handle to match line direction. I think it should not be removed.

Reply:
ok to keep it